sdks/python/apache_beam/examples/flink/flink_streaming_impulse.py

```diff
@@ -57,15 +58,24 @@ def run(argv=None):
     args.extend(argv)
 
   parser = argparse.ArgumentParser()
-  _, pipeline_args = parser.parse_known_args(args)
+  parser.add_argument('--count',
+                      dest='message_count',
+                      default=10000,
+                      help='Number of records to generate.')
+  parser.add_argument('--interval_ms',
+                      dest='interval_ms',
+                      default=500,
+                      help='Interval between records.')
+
+  known_args, pipeline_args = parser.parse_known_args(args)
 
   pipeline_options = PipelineOptions(pipeline_args)
 
   p = beam.Pipeline(options=pipeline_options)
 
   messages = (p | FlinkStreamingImpulseSource()
-              .set_message_count(10000)
-              .set_interval_ms(500))
+              .set_message_count(known_args.message_count)
```

Comment:
I didn't have time to comment on the original PR, but the pythonic way to do this would be to let the constructor take keywords arguments rather than have builder functions.

Reply:
Thanks, will keep that in mind for future work.